apps/api/migrations/059_create_jobs.up.sql

```
@@ -0,0 +1,50 @@
+    updated_at      TIMESTAMPTZ  NOT NULL DEFAULT NOW(),
+
+    CONSTRAINT jobs_status_check
+        CHECK (status IN ('pending', 'running', 'succeeded', 'failed', 'dead')),
+    CONSTRAINT jobs_attempts_nonneg CHECK (attempts >= 0),
+    CONSTRAINT jobs_max_attempts_pos CHECK (max_attempts >= 1)
+);
+
+-- Dequeue hot path: fetch due `pending` rows highest-priority, oldest-first.
+CREATE INDEX idx_jobs_pending_ready
+    ON jobs (priority DESC, next_run_at)
+    WHERE status = 'pending';
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)